homework/04LessonFour.js

```diff
@@ -76,6 +91,9 @@ var nestedObjects = {
     }
   }
 };
+
+nestedObjects["users"]["bob roberts"]["starring"] = 'Tim Robbins';
```

Comment:
Good use of bracket notation!

![celebrate](https://media.giphy.com/media/l46CdUTy21h8wwrAs/giphy.gif)